stock_location_analytic_change_product_qty/README.rst

```diff
@@ -0,0 +1,16 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Stock Analytic
+==============
+
+ When updating the product qty referring to an analytic account the location
+ must contain the same analytic account
+
+
+Usage
+=====
+
+Update the product qty as always
```

Comment:
Below add
* Aaron Henriquez <ahenriquez@eficent.com> and yourself as contributors